dask/bytes/core.py

```diff
@@ -281,64 +223,66 @@ class OpenFile(object):
 
     Parameters
     ----------
-    myopen: function
-        Opens the backend file. Should accept path and mode, as the builtin open
-    path: str
+    fs : FileSystem
+        The file system to use for opening the file
+    path : str
         Location to open
-    compression: str or None
-        Compression to apply
-    mode: str like 'rb'
+    mode : str like 'rb', optional
         Mode of the opened file
-    text: bool
+    compression : str or None, optional
+        Compression to apply
+    text : bool, optional
         Whether to wrap the file to be text-like
-    encoding: if using text
-    errors: if using text
+    encoding : str or None, optional
+        The encoding to use if opened in text mode.
+    errors : str or None, optional
+        How to handle encoding errors if opened in text mode.
     """
-    def __init__(self, myopen, path, compression, mode, text, encoding,
+    def __init__(self, fs, path, mode='rb', compression=None, encoding=None,
```

Comment:
Is it safe to change to order here? I don't know if any code now calls this without keywords, but it's possible.

Reply:
This is only called internally, and all occurrences are changed. The argument values are significantly different now, so none of this is backwards compatible.